src/events/client/interactionCreate.ts

```diff
@@ -104,6 +104,7 @@ export default async function interactionCreateHandler(
                 logger.error(
                     `No handler found for CHAT_INPUT CommandInteraction: ${interaction.data.name}`
                 );
+                return;
```

Comment:
Are the early returns needed for `No handler found` cases? We already return if `interactionPromise === null`